src/commands/Testing/setxp.ts

```diff
@@ -0,0 +1,25 @@
+import { KlasaMessage, CommandStore } from 'klasa';
+import { BotCommand } from '../../lib/BotCommand';
+
+export default class extends BotCommand {
+	public constructor(store: CommandStore, file: string[], directory: string) {
+		super(store, file, directory, {
+			usage: '<amount:int{1,200000000}> <skillName:string>',
```

Comment:
this needs protection against wrong skill names being inputted, in some way. Maybe check that `Skills.some(skill => skill.id === skillName.toLowerCase())` or something like that